src/main/java/net/sf/mzmine/util/maths/similarity/SpectraSimilarity.java

```diff
@@ -55,6 +54,7 @@ public double getCosine() {
 
 
   /**
+   * Cosine similarity

```

Comment:
SpectraSimilarity

Create similarity object for two scans (masslist -> DataPoint[])